hera_cal/utils.py

```diff
@@ -497,80 +502,78 @@ def JD2LST(JD, longitude=21.42830):
         _array = False
         JD = [JD]
 
-    # iterate over JD
-    LST = []
-    for jd in JD:
-        # construct astropy Time object
-        t = Time(jd, format='jd', scale='utc')
-        # get LST in radians at epoch of jd
-        LST.append(t.sidereal_time('apparent', longitude=longitude * unt.deg).radian)
-    LST = np.array(LST)
+    # use pyuvdata
+    LST = uvutils.get_lst_for_time(np.array(JD), latitude, longitude, altitude)
 
     if _array:
         return LST
     else:
         return LST[0]
 
 
-def LST2JD(LST, start_jd, longitude=21.42830):
+def LST2JD(LST, start_jd, allow_other_jd=False, latitude=-30.721526120689507,
+           longitude=21.428303826863015, altitude=1051.690000018105):
     """
     Convert Local Apparent Sidereal Time -> Julian Date via a linear fit
     at the 'start_JD' anchor point.
 
     Input:
     ------
-    LST : type=float, local apparent sidereal time [radians]
+    LST : type=float or array-like, local apparent sidereal time [radians]
 
     start_jd : type=int, integer julian day to use as starting point for LST2JD conversion
+    
+    allow_other_jd : treat the LST at the very beginning of the start_jd as the phase wrap 
```

Comment:
bool type

Reply:
and I don't really understand what this is saying--is there a more straightforward way to explaining it? :)
what do you mean as 'phase wrap reference'. If LST is at the beginning, then it wouldn't wrap to previous JDs, only future JDs, no?

Reply:
I think I've made this clearer now.